spec/latest/core/index.html

```diff
@@ -560,6 +560,17 @@ <h3>Documenting a Web API</h3>
       <i>ApiDocumentation</i>.</p>
 
     <p class="issue">Describe the various properties of an operation.</p>
+
+    <p>Hydra allows also to document both <i>ApiDocumentation</i> and
+      hypermedia controls with human-readable descriptions by applying
+      <i>hydra:title</i> and <i>hydra:description</i> (as shown in the
```

Comment:
I think we decided a while back to exclude the `hydra:` prefix in examples

```suggestion
      <i>title</i> and <i>description</i> (as shown in the
```